src/components/features/accordion/alis/Object.vue

```diff
@@ -0,0 +1,199 @@
+const cadastralId = computed(() => props.getCadastralId(feature.value));
+const hasCadastralId = computed(() => !!cadastralId.value);
+
+function isLoading(activeTab: string) {
+  if (activeTab === "info") {
+    return !!alisWaterBodyLoading.value;
+  } else if (activeTab === "fishstocking") {
+    return !!zuvinimasInfoLoading.value;
+  }
+
+  return false;
+}
+
+const { data: alisWaterBody, isFetching: alisWaterBodyLoading } = useFetch<any>(
+  getWaterBodyInfoUrl(cadastralId.value),
+  { immediate: hasCadastralId.value }
+).json();
+
+const { data: zuvinimasInfo, isFetching: zuvinimasInfoLoading } = useFetch<any>(
+  getFishTypesInfoByYearUrl({ cadastralId: cadastralId.value }),
+  {
+    immediate: hasCadastralId.value,
+    afterFetch(ctx: any) {
+      ctx.data = ctx.data[cadastralId.value] || {};
+      return ctx;
+    },
+  }
+).json();
+
+const tabs = [
+  { type: "info", name: "Bendra info" },
+  { type: "fishstocking", name: "Žuvų įveisimas" },
+];
+
+function getFeatureProp(name: string | string[]) {
+  return props.getValue(feature.value, name);
+}
+const basicInfo = [
+  { key: "Pavadinimas", value: () => getFeatureProp("Pavadinimas") },
+  {
+    key: "UETK kadastro ID",
+    value: () => cadastralId.value,
+  },
+  { key: "Kategorija", value: () => props.getCategory(feature.value) },
+  { key: "Savivaldybė", value: () => alisWaterBody.value?.savivaldybe?.pavadinimas },
+  {
+    key: "Centro X koordinatė (LKS-94), m",
+    value: () => getFeatureProp("Centro X koordinatė (LKS-94), m"),
+  },
+  {
+    key: "Centro Y koordinatė (LKS-94), m",
+    value: () => getFeatureProp("Centro Y koordinatė (LKS-94), m"),
+  },
+  {
+    key: "Žiočių X koordinatė (LKS-94), m",
+    value: () => getFeatureProp("Žiočių X koordinatė (LKS-94), m"),
+  },
+  {
+    key: "Žiočių Y koordinatė (LKS-94), m",
+    value: () => getFeatureProp("Žiočių Y koordinatė (LKS-94), m"),
+  },
+  {
+    key: "Vandens paviršiaus be salų plotas, ha",
+    value: () => getFeatureProp("Vandens paviršiaus be salų plotas, ha"),
+  },
+  {
+    key: "Kranto linijos ilgis, km",
+    value: () => getFeatureProp("Kranto linijos ilgis, km"),
+  },
+  {
+    key: "Statusas",
+    value: () =>
+      hasCadastralId.value
+        ? `${alisWaterBody.value?.isnuomotas ? "Išnuomotas" : "Neišnuomotas"} telkinys`
+        : "",
+  },
+  {
+    key: "Limituota žvejyba?",
+    value: () =>
+      hasCadastralId.value ? (alisWaterBody.value?.licencija ? "Taip" : "Ne") : "",
+  },
+  {
+    key: "Google žemėlapis",
+    link: "Nuoroda į Google",
+    value: () => {
+      const y =
+        getFeatureProp("Centro X koordinatė (LKS-94), m") ||
+        getFeatureProp("Žiočių X koordinatė (LKS-94), m");
+      const x =
+        getFeatureProp("Centro Y koordinatė (LKS-94), m") ||
+        getFeatureProp("Žiočių Y koordinatė (LKS-94), m");
+
+      if (!x || !y) return "";
+      const [newY, newX] = convertCoordinates([x, y], projection, projection4326);
```

Comment:
Looks suspicious, because input is `x, y` and output `y, x`. Is this really correct?

Reply:
yes. it looks suspicious but LKS has "inverted" places for X Y compared to WGS.